chapter3.adoc

```diff
@@ -16,42 +16,20 @@ Will be addressed as follows.
 
 The Debug Module Security Extension imposes security constraints and introduces non-backward-compatible changes. The presence of the Debug Module Security Extension can be determined by polling the allsecured/anysecured bits in dmstatus.
 
-[caption="Register {counter:rimage}: ", reftext="Register {rimage}"]
-[title="The `allsecured` and `anysecured` fields in dmstatus"]
-[id=dmstatus]
-[wavedrom, ,svg]
-....
-{reg: [
-  {bits:   20, name: 'defined in Debug Module'},
-  {bits:   1, name: 'allsecured'},
-  {bits:   1, name: 'anysecured'},
-  {bits:   3, name: 'defined in Debug Module'},
-  {bits:   7, name: '0'},
-], config:{lanes: 3, hspace:1024}}
-....
-
-[options="header"]
-|===============================================================================================================================================
-| Field      | Description                                                                                                | Access | Reset
-| allsecured |  The field is 1 when all currently selected hart are secured and not running in debuggable privilege levels | R    | 0    
-| anysecured |  The field is 1 when any currently selected hart is secured and not running in debuggable privilege levels  | R    | 0    
-|===============================================================================================================================================
-
 === Halt
 
 The hart can only be halted when it is running in privilege levels specified in <<dbgpriv>>. Otherwise, the halt request will remain pending. Consequently, the response to the external debugger request may take longer than one second, and this constraint must be eliminated.
 
-When machine mode is not permitted (`mdbgen` set to 0) to engage in debugging, the halt-on-reset (resethaltreq) operation must fail and set cmderr to 6 (security fault error).
```

Comment:
Why has this been removed? Is halt-on-reset now permitted regardless of the state of mdbgen[i]?

Reply:
I mistake it as a variant of reset. Thanks for spotting it! Will bring it back